api/git/git.go

```diff
@@ -33,6 +38,16 @@ func (service *Service) ClonePrivateRepositoryWithBasicAuth(repositoryURL, refer
 }
 
 func cloneRepository(repositoryURL, referenceName string, destination string) error {
+	customClient := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		},
+
+		Timeout: 15 * time.Second,
```

Comment:
Any particular reason for picking a 15 second timeout? Any impact when cloning large repositories?

Reply:
Not really, just picked up the number that was shown on the example. 
2 minutes sound good to you?

Reply:
Yeah 2min sounds good.